infra/scoped/iam.tf

```diff
@@ -60,38 +60,6 @@ resource "aws_iam_role_policy_attachment" "AWSLambdaVPCAccessExecutionRole" {
   policy_arn = "arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole"
 }
 
-# SES
-
-resource "aws_iam_user" "auth0_email" {
```

Comment:
This has moved to the static stack - we don't need a user per env as we only send SES e-mails in production.